template-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -18,14 +18,13 @@ object Versions {
     const val CHUCKER_VERSION = "3.5.2"
     const val COMPOSE_BOM_VERSION = "2022.12.00"
     const val COMPOSE_COMPILER_VERSION = "1.3.2"
-    const val COMPOSE_NAVIGATION_VERSION = "2.5.1"
+    const val COMPOSE_NAVIGATION_VERSION = "2.5.3"
```

Comment:
FYI: we upgraded this Compose Nav version in the sample project in this PR https://github.com/nimblehq/android-templates/pull/390/files#diff-1767b2c72e6c35afc9709a9fc6b8cf51aea5910d3fe8fca47b97a4a5c4db6deaR21 but forgot to upgrade it in template project. So it's good to keep it here.